src/utils/rewriteImports.js

```diff
@@ -17,7 +17,8 @@ export default (error, file, contents, moduleContext, callback) => {
   const rewritten = contents.replace(importRegexp, (entire, single, double, unquoted) => {
     const oldImportPath = single || double || unquoted;
     const absoluteImportPath = path.join(path.dirname(file), oldImportPath);
-    const newImportPath = path.relative(moduleContext, absoluteImportPath);
+    const relImportPath = path.relative(moduleContext, absoluteImportPath);
+    const newImportPath = relImportPath.split(path.sep).join('/');
```

Comment:
@alexfedoseev Please review this. This looks OK in concept to me.

@bjnsn: I don't like 

```js
relImportPath.split(path.sep)
``` 

I'd rather see a more **direct** call to a file system library to split the path into an array. The code above looks like you are splitting on either a `\` or `/`, and what if that character is escaped in the path.